ion/src/simulator/shared/main_sdl.cpp

```diff
@@ -1,24 +1,55 @@
 #include "main.h"
 #include "display.h"
 #include "platform.h"
-#if !EPSILON_SDL_SCREEN_ONLY
 #include "layout.h"
-#endif
 
 #include <assert.h>
+#include <string.h>
 #include <ion.h>
+#include <stdio.h>
 #include <ion/timing.h>
 #include <ion/events.h>
 #include <SDL.h>
 #include <vector>
 
+static bool argument_screen_only = false;
+static bool argument_fullscreen = false;
+static bool argument_unresizable = false;
+
 void Ion::Timing::msleep(uint32_t ms) {
   SDL_Delay(ms);
 }
 
+void print_help(char * program_name) {
+  printf("Usage: %s [options]\n", program_name);
+  printf("Options:\n");
+  printf("  -f, --fullscreen          Starts the emulator in fullscreen\n");
+  printf("  -s, --screen-only         Disable the keyboard.\n");
+  printf("  -u, --unresizable         Disable resizing the window.\n");
+  printf("  -h, --help                Show this help menu.\n");
+}
+
 int main(int argc, char * argv[]) {
   std::vector<const char *> arguments(argv, argv + argc);
 
+  for(int i = 1; i < argc; i++) {
+    if(strcmp(argv[i], "-h")==0 || strcmp(argv[i], "--help")==0) {
+      print_help(argv[0]);
+      return 0;
+    } else if(strcmp(argv[i], "-s")==0 || strcmp(argv[i], "--screen-only")==0) {
+      argument_screen_only = true;
+    } else if(strcmp(argv[i], "-f")==0 || strcmp(argv[i], "--fullscreen")==0) {
+      argument_fullscreen = true;
+    } else if(strcmp(argv[i], "-u")==0 || strcmp(argv[i], "--unresizable")==0) {
+      argument_unresizable = true;
+    }
+  }
```

Comment:
1. Can you please keep formatting consistent with the rest of the code, i.e. space in `for (` and `if (` and spaces around ` == `.
2. Can you please use existing SDL factilities here instead of relying on libc, so `SDL_strcasecmp`, see `ion/src/simulator/external/sdl/src/test/SDL_test_common.c`.